Fix for requests with large POSTS

codewatchorg opened the pull request:
I've fixed an issue reported by Darryn Ten in which long POST bodies resulted in an empty body request to sqlmap due to fixed column and rows within the plugin GUID that were line breaks to the data going through to sqlmap.  The latest version, 0.5.4, incorporates this fix.

darrynten commented:
Hi @codewatchorg this actually doesn't include the fix from #13 - perhaps re-open and merge that PR into this one?

codewatchorg commented:
I am looking at your pull request from #13 and it shows you removed lines #444 and #445. I have my change right next to it (4d0ca42) in which it shows I removed lines #444 and #445 as well.

The only difference is I also removed lines #490 and #491 as it resets the exact same settings (setColumns to 20 and setRows to 5).  I don't understand how the problem isn't fixed if your solution was to remove where the columns and rows get set.
